tests/ulyanov_et_al_2016/test_modules.py

```diff
@@ -76,6 +109,20 @@ def test_ulyanov_et_al_2016_upsample(subtests):
         assert module.mode == "nearest"
 
 
+def test_UlyanovEtAl2016HourGlassBlock(subtests):
+    intermediate = nn.Conv2d(3, 3, 1)
+    sequential = modules.UlyanovEtAl2016HourGlassBlock(intermediate)
```

Comment:
Why `sequential`? Wouldn't `hour_glass` or `hour_glass_block` be a better fit here?